src/generate/renders/generated/CliCommands.ts

```diff
@@ -44,6 +45,19 @@ export class CliCommands extends TemplateBase {
         return `'${str}'`;
     }
 
+    addImport(importKey: string, importValues: string[]) {
+        let list = [];
+        if (this.imports.has(importKey)) {
+            list = this.imports.get(importKey);
+        }
+        importValues.forEach((val) => {
+            if (list.indexOf(val) === -1) {
+                list.push(val);
```

Comment:
Does CLI have the requirement to order all the import values?

Reply:
yes.